### PR TITLE
Docs: Charter and Governance

### PR DESCRIPTION
**Pull request by molant**
This is part of the work requested by the OpenJS Foundation in #3163 and needs approval from the maintainers and maybe OpenJS' CoC

### CODE REVIEW

**Comment by molant** [comment]
@hxlnt @Malvoz can you please provide feedback on these updates? The scope part might need some more work so any editing is more than welcome :)

Thanks!

**Comment by molant** [comment]
Thanks! I'll wait a couple days so @hxlnt and hopefully @antross can take a look before pinging the foundation for review 😊

File: packages/hint/docs/about/GOVERNANCE.md [hunk 1]
```diff
@@ -146,27 +147,29 @@ items and commit to being a part of the community for the long-term.
...
 A nomination will result in discussion and then a decision by the TSC.
 
-#### TSC Meetings
+#### Meetings
 
-There are no recurrent TSC meetings, they are scheduled when required,
-at a time that works for the TSC members, and using tools that enable
-participation by the community. The meeting is run by a designated
-moderator approved by the TSC.
+The webhint team tries to meet every 2 weeks (length of a sprint)
```
Comment (hxlnt) [inline packages/hint/docs/about/GOVERNANCE.md]: This bit about sprint meetings seems outside the scope of governance/TSC. Curious why this was changed, as I would vote for removing or reverted to previous text.

Comment (molant) [reply packages/hint/docs/about/GOVERNANCE.md]: We do try to meet every couple weeks and I believe it's important for the community to know that happens in case they want to join and/or observe. Other projects that mention meetings:

* [eslint](https://github.com/eslint/eslint/blob/d6c112289f0f16ade070865c8786831b7940ca79/docs/maintainer-guide/governance.md#tsc-meetings)
* [babel](https://github.com/babel/notes): There isn't a governance document
* [node.js](https://github.com/nodejs/node/blob/master/GOVERNANCE.md#tsc-meetings)
* [OpenJS CPC](https://github.com/openjs-foundation/cross-project-council/blob/master/GOVERNANCE.md#team-meetings)

Comment (hxlnt) [reply packages/hint/docs/about/GOVERNANCE.md]: Yes, but it looks like all of these are for TSC meetings about governance, not sprint meetings. For example:

> Items are added to the Team agenda that are considered contentious or are modifications of governance, contribution policy, Team membership, or release process.
> The intention of the agenda is not to approve or review all patches; that should happen continuously on GitHub and be handled by the larger group of Collaborators.

Comment (molant) [reply packages/hint/docs/about/GOVERNANCE.md]: > **Babel:** To setup, we make a PR on babel/notes and then setup a doodle for availability each week (and we make a Slack call or Google Hangouts for the chat). Currently, we usually do video chat on Mondays at 10:00 PM EST.
> 
> If we don't meet, I usually post a summary of what's been going on (notable issues/PRs, etc).

The OpenJS one doesn't have a TSC and it's open to anyone to participate (and the document explains how/where it is run). We don't review PRs in those meetings. We've been doing planning and coordination with people not in the TSC. It is important for anyone interested to know how/when the team decides to work on something next (and even participate).

File: packages/hint/docs/about/PROJECT_CHARTER.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+## 2. Scope
+
+Webhint's goal is to help web developers create better websites by
+providing feedback and best practices guidance in several areas such
+as (but not limited to) accessibility, security, performance, etc.
+
+Webhint should be useful during all the development lifecycle and
+adapt its feedback to each stage and the users needs: it can be run
+as a CLI, VS Code extension, browser extension and as an online
+service.
```
Comment (hxlnt) [inline packages/hint/docs/about/PROJECT_CHARTER.md]:
```suggestion
adapt its feedback to each stage and the users' needs.
```

File: packages/hint/docs/about/GOVERNANCE.md (hunk 1 above)
Comment (hxlnt) [reply packages/hint/docs/about/GOVERNANCE.md]: OK.

File: packages/hint/docs/about/PROJECT_CHARTER.md (hunk 2 above)
Comment (hxlnt) [reply packages/hint/docs/about/PROJECT_CHARTER.md]: Recommending this so that we don't need to update this as product suite based on webhint evolves.

**Comment by molant** [comment]
I've talked with @brianwarner

 > I’d recommend merging it as GOVERNANCE.md-draft and CHARTER.md-draft and opening an issue against those.  Effectively by merging, you’re saying “the project is in agreement on this text, and it’s on to the next phase”

So as soon as I can (or @hxlnt if you have time) I'll take care of the renaming. The next step will be to open an issue in https://github.com/openjs-foundation/cross-project-council/issues asking to review these files.

**Comment by molant** [comment]
Opened https://github.com/openjs-foundation/cross-project-council/issues/482
